dist/Dockerfile.build/Dockerfile

```diff
@@ -11,7 +11,8 @@ ENV HOME="/root"
 ENV PATH="${HOME}/.cargo/bin:${PATH}"
 
 # build: Rust stable toolchain
-RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.46.0 -y
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.46.0 -y && \
+  rustup install 1.43.1
```

Comment:
Why do we need to install rust 1.43.1 explicitly? Just to maintain backward compatibility

Reply:
Yes.